script.js

```diff
@@ -0,0 +1,75 @@
+// Task 1
+for (let i = 10; i <= 25; i += 0.5) {
+  console.log(i);
+}
+
+//Task 2
+let askingNumber = prompt("Please enter prime number");
+while (askingNumber % 2 == 0) {
+  alert("Your number is not prime");
+  askingNumber = prompt("Please enter number");
+}
+alert("Your number is prime");
+
+//Task 3
+lowDiscount = 0.03;
+mediumDiscount = 0.05;
+highDiscount = 0.07;
+currency = "UAH";
+const SumOforder = prompt("Sum of order?");
+
+if (SumOforder <= 100) {
+  alert(`You have discount : ${(SumOforder * lowDiscount) .toFixed(2)} ${currency}`);
```

Comment:
We are trying to avoid using `alert` since it blocks the execution of JS below and page is freezed